src/main/java/frc/robot/subsystems/intake/IntakeIOTalonFX.java

```diff
@@ -0,0 +1,250 @@
+    inputs.rollerVelocityRPS =
+        rollerVelocityStatusSignal.getValueAsDouble();
+    inputs.kickerVelocityRPS = 
+        kickerVelocityStatusSignal.getValueAsDouble();
+
+    inputs.rollerReferenceVelocityRPS =
+        rollerReferenceVelocityStatusSignal.getValueAsDouble();
+    inputs.kickerReferenceVelocityRPS = 
+        kickerReferenceVelocityStatusSignal.getValueAsDouble();
+
+    if (rollerMotorsKP.hasChanged()
+        || rollerMotorsKI.hasChanged()
+        || rollerMotorsKD.hasChanged()
+        || rollerMotorsKS.hasChanged()) {
+      
+      Slot0Configs slot0Configs = new Slot0Configs();
+      rollerMotor.getConfigurator().refresh(slot0Configs);
+      slot0Configs.kP = rollerMotorsKP.get();
+      slot0Configs.kI = rollerMotorsKI.get();
+      slot0Configs.kD = rollerMotorsKD.get();
+      slot0Configs.kS = rollerMotorsKS.get();
+
+      rollerMotor.getConfigurator().apply(slot0Configs);
+    }
+
+    if (kickerMotorKP.hasChanged()
+        || kickerMotorKI.hasChanged()
+        || kickerMotorKD.hasChanged()
+        || kickerMotorKS.hasChanged()) {
+      Slot0Configs slot0Configs = new Slot0Configs();
+      kickerMotor.getConfigurator().refresh(slot0Configs);
+      slot0Configs.kP = rollerMotorsKP.get();
+      slot0Configs.kI = rollerMotorsKI.get();
+      slot0Configs.kD = rollerMotorsKD.get();
+      slot0Configs.kS = rollerMotorsKS.get();
+
+      kickerMotor.getConfigurator().apply(slot0Configs);
+    }
+  }
+
+  @Override
+  public void setRollerVelocity(double rps) {
+    rollerMotor.setControl(rollerVelocityRequest.withVelocity(rps));
+  }
+
+  @Override
+  public void setKickerVelocity(double rps) {
+    kickerMotor.setControl(kickerVelocityRequest.withVelocity(rps));
+  }
+
+  private void configureIntakeRollerMotor(TalonFX rollerMotor) {
+    TalonFXConfiguration rollerConfig = new TalonFXConfiguration();
+    CurrentLimitsConfigs rollerCurrentLimits = new CurrentLimitsConfigs();
+
+    rollerCurrentLimits.SupplyCurrentLimit = IntakeConstants.ROLLERS_CONTINUOUS_CURRENT_LIMIT;
+    rollerCurrentLimits.SupplyCurrentThreshold = IntakeConstants.ROLLERS_PEAK_CURRENT_LIMIT;
+    rollerCurrentLimits.SupplyTimeThreshold = IntakeConstants.ROLLERS_PEAK_CURRENT_DURATION;
+    rollerCurrentLimits.SupplyCurrentLimitEnable = true;
+
+    rollerConfig.CurrentLimits = rollerCurrentLimits;
+
+    rollerConfig.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+
+    rollerConfig.Slot0.kP = rollerMotorsKP.get();
+    rollerConfig.Slot0.kI = rollerMotorsKI.get();
+    rollerConfig.Slot0.kD = rollerMotorsKD.get();
+    rollerConfig.Slot0.kS = rollerMotorsKS.get();
+
+    rollerConfig.MotorOutput.Inverted =
+        IntakeConstants.RIGHT_ROLLER_MOTOR_INVERTED
+            ? InvertedValue.Clockwise_Positive
+            : InvertedValue.CounterClockwise_Positive;
+
+    rollerConfig.Feedback.SensorToMechanismRatio =
+        IntakeConstants.ROLLERS_SENSOR_TO_MECHANISM_RATIO;
+
+    rollerMotor.getConfigurator().apply(rollerConfig);
```

Comment:
Use this code from SubsystemIOTalonFX.java to make sure the configuration is applied:

```
StatusCode status = StatusCode.StatusCodeNotInitialized;
    for (int i = 0; i < 5; ++i) {
      status = this.motor.getConfigurator().apply(config);
      if (status.isOK()) {
        configAlert.set(false);
        break;
      }
    }
    if (!status.isOK()) {
      configAlert.set(true);
      configAlert.setText(status.toString());
    }
```

(also do this for the other config method)